### ISSUE TITLE
Up and down height direction is counterintuitive

### ISSUE THREAD

**Issue by jvolker** [nornagon/saxi#59]
I think the up and down height direction is counterintuitive. The up position should have a higher number compared to the down position. Increasing the numbers should lift the pen up, not down.

Would be great to invert those numbers. Thanks.

**Comment by nornagon** [comment]
I disagree :)

**Comment by jvolker** [comment]
Why?

**Comment by jvolker** [comment]
Is there a reason? Did you implement this from a technical perspective, do you think it's more intuitive for an average user or is this simply your personal preference?

I'm a user who has recently started using plotters.

Thanks.

**Comment by nornagon** [comment]
This matches the way the firmware works.

I think about it as _how "down" is the pen, on a scale from 0% to 100%_.

Now that I look, it seems like the Inkscape driver maybe has things the other way around, which is a reasonable reason to think about switching it: https://github.com/evil-mad/axidraw/blob/74b7e800c70a9284d6ff7e182b21603c81c7e945/inkscape%20driver/axidraw_conf.py#L39-L40

**Comment by EmmaSimon** [comment]
Just another perspective, but I was confused about this at first after being familiar with Inkscape.

**Comment by ianstormtaylor** [comment]
Yeah was also confused by this on first use, and have to re-orient whenever I need to adjust the pen height again (which luckily isn't too often).

Either way, thank you for Saxi!

**Comment by alexrudd2** [comment]
+1 on this. I thought it was a bug until finding this explanation. :)
